### ISSUE TITLE
WiFi / ETH issue?

### ISSUE THREAD

**Issue by shawaj** [NebraLtd/helium-miner-software#57]
I've noticed on a few support tickets people having issues with WiFi when they have both WiFi and Ethernet connected.

I wonder if this is a wider issue as I've seen at least 3 or 4 with similar issue. Not got any conclusive evidence to confirm it, was just anecdotal but thought worth mentioning

**Comment by shawaj** [comment]
Maybe related to #21 ?

**Comment by marbletissue** [comment]
I just experienced this issue I believe. I have an open ticket relating to BT and Ethernet connectivity (ID #258255), however I rebooted my hotspot with Ethernet connected and noticed I couldn't connect to either WiFi or Ethernet static IP addresses via the web browser to check the hotspots status (IP addresses have been statically assigned on my router). I disconnected the Ethernet, rebooted and could access the Diagnostics webpage using my WiFi IP address.
